examples/bri-3/src/bri/vsm/capabilites/executeVsmCycle/executeVsmCycleCommand.handler.ts

```diff
@@ -29,8 +29,11 @@ export class ExecuteVsmCycleCommandHandler
       await this.txStorageAgent.updateTransactionStatus(tx);
 
       if (!this.agent.validateTransactionForExecution(tx)) {
-        tx.updateStatusToInvalid();
-        await this.txStorageAgent.updateTransactionStatus(tx);
+        this.eventBus.publish(
+          new WorkstepExecutionFailuresEvent(tx, 'Validation Error'),
+        );
+        tx.updateStatusToAborted();
```

Comment:
tx.updateStatusToInvalid();

Reply:
@ognjenkurtic Fixed. Apologizes for not reading the code carefully.

Reply:
no worries, approved now, looks great